src/usr/local/buildpack/tools/v2/python.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+# sets the correct shebang for python
+fix_python_shebangs() {
+  # https://github.com/koalaman/shellcheck/wiki/SC2044
+  while IFS= read -r -d '' file
+  do
+    case "$(head -1 "${file}")" in
+    "#!"*"/bin/python" )
+      sed -i "1 s:.*:#\!${versioned_tool_path}\/bin\/python:" "${file}"
+      ;;
+    "#!"*"/bin/python${MAJOR}" )
+      sed -i "1 s:.*:#\!${versioned_tool_path}\/bin\/python${MAJOR}:" "${file}"
+      ;;
+    "#!"*"/bin/python${MAJOR}.${MINOR}" )
+      sed -i "1 s:.*:#\!${versioned_tool_path}\/bin\/python${MAJOR}.${MINOR}:" "${file}"
+      ;;
+    esac
+  done < <(find "${versioned_tool_path}/bin" -type f -exec grep -Iq . {} \; -print0)
+}
+
+function prepare_tool() {
```

Comment:
i think we should do a major version bump and drop this. so we no longer need any prepare step.

will do this in a separate PR today before this PR.